packages/streams/execute.js

```diff
@@ -116,8 +117,10 @@ Execute.prototype.pipe = function (pipeTo) {
   })
 
   if (stdOutEvent === 'nmea2000out') {
-    that.options.app.on('nmea2000outJson', pgn => {
-      that.serial.write(pgnToActisenseSerialFormat(pgn) + '\r\n')
+    that.options.app.on('nmea2000JsonOut', pgn => {
+      console.log('n2k json')
```

Comment:
Remove the `console.log`'s?

Reply:
Lol, hard to understand how I managed not to see them. Fixed.